tahrir/views.py

```diff
@@ -1437,6 +1437,7 @@ def report_year_month(request):
     year = int(request.matchdict.get('year'))
     month = int(request.matchdict.get('month'))
 
+    now = datetime.now()
```

Comment:
It is only using the date part of it, so not sure it matters... Also, our servers are all on UTC.